shell/src/main/java/alluxio/cli/fs/command/CopyFromLocalCommand.java

```diff
@@ -60,11 +66,11 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "copyFromLocal <src> <remoteDst>";
+    return "copyFromLocal [-t <number of threads for copying>] <src> <remoteDst>";
   }
 
   @Override
   public String getDescription() {
-    return "Copies a file or a directory from local filesystem to Alluxio filesystem.";
+    return "Concurrently copies a file or a directory from local filesystem to Alluxio filesystem.";
```

Comment:
Can we clarify how this is concurrent? It currently sounds like a single file can be copied concurrently, but that isn't the case, right?

Reply:
changed to `Copies .... in parallel at file level.`